src/ukify/ukify.py

```diff
@@ -927,6 +929,16 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
         config_key = 'UKI/Stub',
     ),
 
+    ConfigItem(
+        '--sbat',
+        metavar = 'TEXT|@PATH',
+        help = 'SBAT policy [.sbat section] for addons',
+        default = """sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+uki.addon.systemd,1,UKI Addon,uki.addon.systemd,1,https://www.freedesktop.org/software/systemd/man/systemd-stub.html
+""",
```

Comment:
The distro vendor should probably be in the default too.

Reply:
mmh for addons? I don't think so? This is supposed to be a default for users manually calling this, distro vendors building addons should override completely with their own data

Reply:
i'd drop the reference to systemd really, it's nothing we own.

i.e. `uki-add-on,1,UKI Add-On,uki-add-on,1,https://…`

or so?

btw, "addon"? or "add-on"?

Reply:
It should be 3, actually. "sbat", "systemd" and then "add-on" (which takes the role of vendor/distro). Just like we generate for sd-boot/stub.

Also, I think it's add-on, similar to "log in".

Reply:
I can see it used both as add-on and addon, and I like it more without the hyphen :-)

> It should be 3, actually. "sbat", "systemd" and then "add-on" (which takes the role of vendor/distro). Just like we generate for sd-boot/stub.

The SBAT folks are saying that lines here need to identify code that could be revoked. In the case of an addon, there's no systemd code now as it's just an empty function, so it should not mention the same line that goes in sd-stub or sd-boot.

Reply:
i still think we should add a separate line. There's value in being able to revoke the whole add-on concept I think or different versions of the add-on concept, if we made a mistake. I'd really not outrule that. It's so cheap to add precise info, hence let's add that. 

And yes, I'd still generate distincting lines for add-ons, sd-boot, sd-stub simply so that we can revoke them separately. Would really suck if a faulty old UKI also means we have to revoke an old sd-boot from the same era that didn't fuck up. Because invalidating a kernel is usually not the biggie, you might still have a newer one around, by invalidating the boot loader might mean you cannot boot at all anymore. And that sucks.

Hence, I see no reason why we would be conservative here: let's add different sbat for the three components we provide so that they can be revoked independently.

Reply:
If a UKI is invalidated because of sd-stub, then in all likelihood sd-boot would have the same problem - that's the point. Invalidating UKIs because of kernel issues would be done via the kernel-specific line, not the systemd one, so sd-boot will not be affected.

Reply:
(yes that means we need to extract the kernel sbat and add it to the uki when assembling, but I'm not going to add more stuff to this already large PR)